navigation/mapping_exploration/launch/explore_lite.launch

```diff
@@ -0,0 +1,16 @@
+<launch>
+<node pkg="explore_lite" type="explore" respawn="true" name="explore" output="screen">
+  <param name="robot_base_frame" value="create1_tf/base_link"/>
```

Comment:
As it's being done in order launch files, use arguments to get the robot ID and, from there, create the Tf prefix and namespace.

Reply:
Done too!